nixos/modules/services/x11/hardware/libinput.nix

```diff
@@ -163,6 +163,15 @@ let cfg = config.services.xserver.libinput;
           '';
       };
 
+      transformationMatrix = mkOption {
+        type = types.nullOr types.str;
```

Comment:
maybe this can be
```nix
types.nullOr (types.listOf types.float)
```

Reply:
It could be, but it would be inconsistent with other options in the module (see for example calibrationMatrix). I think if we want to do it like this then calibrationMatrix should be changed as well.

But also, this option is not well documented upstream and it's possible doing it this way would prevent setting some legitimate values. At least I would prefer if someone more familiar with libinput would weigh in before changing this.